src/norm/postgres.nim

```diff
@@ -433,7 +433,7 @@ template transaction*(dbConn; body: untyped): untyped =
     log(commitQry)
     dbConn.exec(sql commitQry)
 
-  except:
+  except CatchableError:
```

Comment:
Does CatchableError cover all cases? You want your transactions rolled back if anything bad happens.

Reply:
It catches any error you should be able to catch. It does not catch defects, but with panics:on, which is what you should be compiling with as that is how they are designed, defects are uncatchable. They are the "root"-error type, if you will.

DBError: CatchableError --> IOError --> DBError
OSError: CatchableError --> OSError